.github/workflows/release-pages.yml

```
@@ -0,0 +1,186 @@
+name: Deploy to GitHub Pages
+description: Build and deploy to GitHub Pages and Vercel on successful completion of tests and builds
```

actionlint (1.7.10)

[error] unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)